wicker/schema/schema.py

```diff
@@ -404,6 +404,147 @@ def __init__(
         )
 
 
+class VariantField(SchemaField):
```

Comment:
The introduce of the schema is for clear data type definition, so personally dont think we should add a new VariantField